src/services/fileStorage/strategies/awsS3.js

```diff
@@ -1,9 +1,9 @@
 const { promisify } = require('es6-promisify');
 const errors = require('@feathersjs/errors');
 const aws = require('aws-sdk');
-const { posix: pathUtil } = require('path');
-const logger = require('../../../logger');
+const pathUtil = require('path');
```

Comment:
only used in an deprecated function, maybe remove all that, I will create a ticket for it.